justfile

```diff
@@ -15,6 +15,6 @@ docs-config := base-directory / "docs/mkdocs.yml"
 serve-docs port="8000":
     poetry run mkdocs serve --config-file {{ docs-config }} --dev-addr localhost:{{ port }}
 
-# Generate CLI Usage page in documentation
-generate-docs:
-    sh ./gen_docs.sh
```

Comment:
@sujuka99 I couldn't find this script anymore, and I deleted it. You added this command 4 months ago:
https://github.com/bakdata/kpops/commit/49beec7f2a09755959eafba170a0a1e8a779c476
Is it okay to delete it?

Reply:
I always had problems with that script xD (windows...)

Reply:
@raminqaf I completely forgot about that! Yes, please delete it.

This script used to generate the "CLI Usage" section, but this was moved months ago to the python hook.